src/ResourceManagement/Compute/ComputeManagement/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.11.0-Nightly20150907
+set autoRestVersion=0.11.0-Nightly20150911
```

Comment:
I check the other folder, and it seems this is the latest version other component APIs use. So, we would also use this one.